site/docs/hosting/cloudflare-workers.md

```diff
@@ -1,94 +1,116 @@
-Having the ability to handle HTTP traffic and being based on the [Service Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), building Telegram bots becomes a breeze.
-In addition, you can even develop [Web Apps](https://core.telegram.org/bots/webapps) at the edge, all for free within certain quotas.
+# Hosting: Cloudflare Workers (Deno)
 
-This guide will take you through the process of hosting your Telegram bots on Cloudflare Workers.
+[Cloudflare Workers](https://workers.cloudflare.com) is a public serverless computing platform that offers a convenient and simple solution for running small workloads at the [edge](https://en.wikipedia.org/wiki/Edge_computing).
+
+This guide will take you through the process of hosting your bot on Cloudflare Workers.
+
+::: tip Looking for the Node.js Version?
+This tutorial explains how to deploy a Telegram bot to Cloudflare Workers using Deno.
+If you're looking for the Node.js version, please check out [this tutorial](./cloudflare-workers-nodejs) instead.
+:::
 
 ## Prerequisites
 
 To follow along, please make sure that you have a [Cloudflare account](https://dash.cloudflare.com/login) with your workers subdomain [configured](https://dash.cloudflare.com/?account=workers).
 
 ## Setting Things Up
 
-First, create a new project:
-
-```sh
-npx wrangler generate my-bot
-```
+Make sure you have [Deno](https://deno.land) and [Denoflare](https://denoflare.dev) installed.
 
-You can change `my-bot` to whatever you want.
-This will be the name of your bot and the project directory.
+Create a new directory, and create a new file `.denoflare` in that directory.
+Put the following contents in the file:
 
-After running the above command, follow the instructions you see to initialize the project.
-There, you can choose between JavaScript or TypeScript.
+> Note: The "$schema" key in the following JSON code specifies a fixed version in its URL ("v0.5.12").
+> At the time of writing, this was the latest version available.
+> You should update them to the [newest version](https://github.com/skymethod/denoflare/releases).
 
-When the project is initialized, `cd` into `my-bot` or whatever directory you initialized your project in.
-Depending on how you initialized the project, you should see a file structure similar to the following:
-
-```asciiart:no-line-numbers
-.
-├── node_modules
-├── package.json
-├── package-lock.json
-├── src
-│   ├── index.js
-│   └── index.test.js
-└── wrangler.toml
+```json{2,9,17-18}
+{
+  "$schema": "https://raw.githubusercontent.com/skymethod/denoflare/v0.5.12/common/config.schema.json",
+  "scripts": {
+    "my-bot": {
+      "path": "bot.ts",
+      "localPort": 3030,
+      "bindings": {
+        "BOT_TOKEN": {
+          "value": "YOUR_BOT_TOKEN"
```

Comment:
I suggest for things that the user should fullfil after, to put like `<BOT_TOKEN>` with `< >`

Also, If we put only the name of the variable, we don´t need to translate to other languages, will be more consistent